_topics/en/transaction-bloom-filtering.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Transaction bloom filtering
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BIP37
+  - Bloom filters
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightweight Client Support
+  - Privacy Problems
+  - P2P Network Protocol
+  - Security Problems
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Transaction bloom filtering** is a method that allows lightweight
+  clients to limit the amount of transaction data they receive from full
+  nodes to only those transaction that affect their wallet (plus a
+  configurable amount of additional transactions for camouflage).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  [Bloom filters][] provide the ability to create a compact filter that
+  is guarantee to match a specified string with a configurable rate of
+  false-positive matches for other strings.  A lightweight client can
+  create a bloom filter for all of its wallet addresses, send that
+  filter to a node using P2P protocol messages defined in BIP37, and
+  then request a special form of blocks (merkle blocks) from the node.
+
+  A merkle block, also defined in BIP37, will contain only transactions
+  matching the previously sent filter plus the block header and a
+  partial merkle branch connecting each matching transaction to the
+  merkle root in the block header.
+
+  Clients will also receive announcements of new transactions being
+  relayed by the node if they match the filter.
+
+  When BIP37 was popular, most lightweight clients that used it ran on
+  mobile devices with limited bandwidth and so chose low false-positive
+  rates to minimize their bandwidth use.  This meant that they
+  essentially gave their list of addresses to any node they contacted.
+  Higher false positive rates may have partially mitigated this privacy
+  leak but false positives would be created at random from the set of
```

Comment:
Careful of shifting verb tense, previously was in the past, here it is conditional, and in the following paragraph shifts again to present, then shifts in the last sentence "This resulted..." back to past tense.